### PR TITLE
Hotovo feature/comments management

### PR DESCRIPTION
**Pull request by coolboy**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @coolboy has signed the CLA. Thanks for the pull request!

File: Box.V2/Models/Request/BoxCommentRequest.cs [hunk 1]
```diff
@@ -14,10 +14,16 @@ public class BoxCommentRequest
...
-        /// The message
+        /// The text body of the comment
         /// </summary>
         [JsonProperty(PropertyName = "message")]
         public string Message { get; set; }
 
+        /// <summary>
+        /// The text body of the comment, including @[userid:Username] somewhere in the message to mention the user, which will send them a direct email, letting them know they’ve been mentioned in a comment.
+        /// </summary>
+        [JsonProperty(PropertyName = "tagged_message")]
```
Comment (coolboy) [inline Box.V2/Models/Request/BoxCommentRequest.cs]: @cburnette I checked on our public doc, the request need either message or tagged_message to be there.

I would question our API design for adding this tagged_message, which we should able to parse @[userid:Username] just from message

File: Box.V2/Managers/BoxCommentsManager.cs [hunk 2]
```diff
@@ -60,8 +65,10 @@ public BoxCommentsManager(IBoxConfig config, IBoxService service, IBoxConverter
         /// <summary>
         /// Used to update the message of the comment.
         /// </summary>
-        /// <param name="id"></param>
-        /// <returns></returns>
+        /// <param name="id">Id of the comment.</param>
+        /// <param name="commentsRequest">commentsRequest.Message (Required) - The desired text for the comment message.</param>
```
Comment (cburnette) [inline Box.V2/Managers/BoxCommentsManager.cs]: yeah change it

Comment (coolboy) [inline Box.V2/Managers/BoxCommentsManager.cs]: done

**Comment by cburnette** [comment]
@coolboy looks good, merging

**Pull request merged**